src/utils/queue/wrap.ts

```diff
@@ -0,0 +1,24 @@
+import { Queue } from 'bullmq'
+
+export type TokenUpdateQueue = ReturnType<typeof wrapTokenQueue>
+export type DeploymentUpdatedQueue = ReturnType<
+  typeof wrapDeploymentUpdatedQueue
+>
+
+// Thin abstraction over the queue so we explicitly assign the job id to
```

Comment:
If this applies to both function (which I guess it does), we should leabe this commend also on `wrapDeploymentUpdatedQueue`

Reply:
https://github.com/l2beat/token-db/pull/31/commits/14331efa99390b178f85dd98055e2119019033ee